zeppelin-server/pom.xml

```diff
@@ -43,19 +43,19 @@
       <dependency>
         <groupId>org.scala-lang</groupId>
         <artifactId>scala-library</artifactId>
-        <version>2.10.4</version>
+        <version>2.11.8</version>
```

Comment:
Here's the "ps aux" I got running zeppelin.sh. It looks like interpreter.sh is picking up ZEPPELIN_CLASSPATH from the server process. Am I doing something wrong?


```
tw-172-25-130-178 incubator-zeppelin-prasadwagle (ZEPPELIN-840) $ ps aux | grep zeppelin
pwagle           91376   0.1  1.0  5221060 159804 s001  S    11:21AM   0:21.61 /Library/Java/JavaVirtualMachines/jdk1.8.0_65.jdk/Contents/Home/bin/java -Dfile.encoding=UTF-8 -Xms1024m -Xmx1024m -XX:MaxPermSize=512m -Dlog4j.configuration=file:///Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/conf/log4j.properties -Dzeppelin.log.file=/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/logs/zeppelin-pwagle-tw-172-25-130-178.office.twttr.net.log -cp :.:/usr/local/lib/antlr-4.0-complete.jar::/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-server/target/lib/*:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-zengine/target/lib/*:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-interpreter/target/lib/*:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/*::/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/conf:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-interpreter/target/classes:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-zengine/target/classes:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-server/target/classes org.apache.zeppelin.server.ZeppelinServer
pwagle           91390   0.1  2.8  5105608 465152 s001  S    11:21AM   0:21.49 /Library/Java/JavaVirtualMachines/jdk1.8.0_65.jdk/Contents/Home/bin/java -Dfile.encoding=UTF-8 -Dlog4j.configuration=file:///Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/conf/log4j.properties -Dzeppelin.log.file=/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/logs/zeppelin-interpreter-scalding-pwagle-tw-172-25-130-178.office.twttr.net.log -Xms1024m -Xmx1024m -XX:MaxPermSize=512m -cp :.:/usr/local/lib/antlr-4.0-complete.jar::/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-server/target/lib/*:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-zengine/target/lib/*:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-interpreter/target/lib/*:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/*::/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/conf:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-interpreter/target/classes:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-zengine/target/classes:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-server/target/classes:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/interpreter/scalding/*:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-interpreter/target/lib/*::/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/conf:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/conf:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-interpreter/target/classes org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer 63581
pwagle           92022   0.1  0.6  5020016  96164 s001  S    12:18PM   0:01.18 /Library/Java/JavaVirtualMachines/jdk1.8.0_65.jdk/Contents/Home/bin/java -Dfile.encoding=UTF-8 -Dlog4j.configuration=file:///Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/conf/log4j.properties -Dzeppelin.log.file=/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/logs/zeppelin-interpreter-md-pwagle-tw-172-25-130-178.office.twttr.net.log -Xms1024m -Xmx1024m -XX:MaxPermSize=512m -cp :.:/usr/local/lib/antlr-4.0-complete.jar::/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-server/target/lib/*:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-zengine/target/lib/*:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-interpreter/target/lib/*:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/*::/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/conf:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-interpreter/target/classes:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-zengine/target/classes:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-server/target/classes:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/interpreter/md/*:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-interpreter/target/lib/*::/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/conf:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/conf:/Users/pwagle/workspace/zeppelin/incubator-zeppelin-prasadwagle/zeppelin-interpreter/target/classes org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer 63923
```